core/src/main/codegen/default_config.fmpp

```diff
@@ -84,8 +84,10 @@ parser: {
     "DATA"
     "DATABASE"
     "DATE_TRUNC"
+    "DATETIME_DIFF"
     "DATETIME_INTERVAL_CODE"
     "DATETIME_INTERVAL_PRECISION"
+    "DATE_DIFF"
```

Comment:
nit: alphabetically next to DATE_TRUNC, or move DATE_TRUNC down here too (visually _ comes before T to me, but maybe ASCII-wise DATE_ should be after DATET?)

Reply:
I was also confused about this, on lines 317-321 of this file, _ is given alphabetic precedence over letters, and then letters over _. I agree that visually it comes before. Maybe @julianhyde has thoughts?

Reply:
I forget what the rules are. I would defer to UNIX `sort` (using `LC_ALL=C` to override my locale):

```
$ LC_ALL=C sort<<EOF
    "DATE_DIFF"
    "DATETIME_INTERVAL_CODE"
    "DATETIME_INTERVAL_PRECISION"
EOF
    "DATETIME_INTERVAL_CODE"
    "DATETIME_INTERVAL_PRECISION"
    "DATE_DIFF"
```